data/ecosystems/c/cardano.toml

```diff
@@ -407,12 +512,6 @@ url = "https://github.com/CardanoUmbrella/awesome-cardano"
 [[repo]]
 url = "https://github.com/cffls/pycardano"
 
-[[repo]]
-url = "https://github.com/cloudstruct/go-cardano-submit-api"
-
-[[repo]]
-url = "https://github.com/cloudstruct/go-ouroboros-network"
```

Comment:
why do you propose to remove these 2 repos?

Reply:
CloudStruct has become BlinkLabs, so I added the BlinkLabs equivalent instead 👍

Reply:
More specifically: 

| Old | New | 
| --- | --- | 
| https://github.com/cloudstruct/go-ouroboros-network | https://github.com/blinklabs-io/gouroboros |
| https://github.com/cloudstruct/go-cardano-submit-api | https://github.com/blinklabs-io/tx-submit-api |

You'll see that Github even redirects you properly if you click the old links.